tools/build/third_party/perfetto/perfetto.BUILD

```diff
@@ -29,7 +29,11 @@ _COPTS_BASE = cc_copts() + [
 })
 
 _COPTS = _COPTS_BASE + [
-    "-DPERFETTO_BUILD_WITH_EMBEDDER",
+    #"-DPERFETTO_BUILD_WITH_EMBEDDER",
```

Comment:
PERFETTO_BUILD_WITH_EMBEDDER doesn't actually do anything anymore and can be removed.

Reply:
So this has knock-on effects. The build still fails without this code.